src/lib/dai.c

```diff
@@ -66,8 +66,7 @@ struct dai *dai_get(uint32_t type, uint32_t index, uint32_t flags)
 		if (!ret)
 			d->sref++;
```

Comment:
s/Fixes/Fix in the commit subject. Also, it might help to paste the exact compiler error in the commit body.